src/biokbase/narrative/tests/test_jobcomm.py

```diff
@@ -301,18 +334,11 @@ def test_req_no_inputs__fail(self):
         }
 
         for msg_type, err_type in expected.items():
-            msg = {
-                "msg_id": "some_id",
-                "content": {
-                    "data": {
-                        "request_type": msg_type,
-                    }
-                },
-            }
+            req = make_comm_msg(msg_type, None, False)
```

Comment:
in a number of places, this manual message structure construction has been replaced by use of the `make_comm_msg` function